autometa/validation/datasets.py

```diff
@@ -0,0 +1,174 @@
+    return targets_dict
+
+
+def download(targets_dict, out_dirpath: str) -> None:
+
+    """Downloads the files specified in a dictionary.
+
+    Parameters
+    ----------
+    targets_dict : dict
+        nested dictionary with the user's specified files' folder, file, and file_id
+        format: {'folder': {<index>: <folder>}, 'file': {<index>: <file>}, 'file_id': {<index>: <file_id>}}
+    out_dirpath : str
+        directory path where the user wants to download the file(s)
+
+    Returns
+    -------
+    None
+        download is completed through gdown
+
+    """
+
+    key_list = [*targets_dict["file"]]
+    for key in key_list:
+        # retrieve values from targets_dict
+        file = targets_dict["file"][key]
+        folder = targets_dict["folder"][key]
+        file_id = targets_dict["file_id"][key]
+
+        # construct file id into a url to put into gdown
+        url = f"https://drive.google.com/uc?id={file_id}"
+        filename = f"{folder}_{file}"
+        out_fpath = os.path.join(
+            out_dirpath, filename
+        )  # this returns an error when relative file paths get used, eg "~/download"
+
+        # download the specified file with gdown
+        gdown.download(url, out_fpath)
+
+
+def main():
+    import argparse
+    import logging as logger
+
+    logger.basicConfig(
+        format="[%(asctime)s %(levelname)s] %(name)s: %(message)s",
+        datefmt="%m/%d/%Y %I:%M:%S %p",
+        level=logger.DEBUG,
+    )
+
+    parser = argparse.ArgumentParser(
+        prog="autometa-download-dataset",
+        description="Download a simulated community file from google drive to a specified directory",
+    )
+    parser.add_argument(
+        "--folder",
+        help="specify a simulated community size to download from (leave blank to download all)",
+        choices=[
+            "78.125Mbp",
+            "156.25Mbp",
+            "312.5Mbp",
+            "625Mbp",
+            "1250Mbp",
+            "2500Mbp",
+            "5000Mbp",
+            "10000Mbp",
+        ],
+    )
+    parser.add_argument(
+        "--file",
+        help="specify a file to download (or leave blank to download all)",
+        choices=[
+            "all",
+            "README.md",
+            "reference_assignments.tsv.gz",
+            "metagenome.fna.gz",
+            "master.tsv.gz",
+            "control_reads.tsv.gz",
+            "control_contigs.tsv.gz",
+            "unclustered_recruitment.tsv.gz",
+            "binning.tsv.gz",
+            "taxonomy.tsv.gz",
+            "lengths.tsv.gz",
+            "coverages.tsv.gz",
+            "gc_content.tsv.gz",
+            "kmers.embedded.tsv.gz",
+            "kmers.tsv.gz",
+            "markers.tsv.gz",
+            "Bacteria.fna.gz",
+            "orfs.faa.gz",
+            "metagenome.filtered.fna.gz",
+            "hmmscan.tsv.gz",
+            "forward_reads.fastq.gz",
+            "reverse_reads.fastq.gz",
+        ],
+    )
```

Comment:
```suggestion
            "reverse_reads.fastq.gz",
        ],
        default="all",
    )
```

Reply:
I went ahead and removed the "all" functionality for now, but I'll see about putting it back in later on.